modules/orca/main.tf

```diff
@@ -31,6 +31,7 @@ module "orca_lambdas" {
   ## REQUIRED
   db_connect_info_secret_arn                           = module.orca_secretsmanager.secretsmanager_arn
   orca_default_bucket                                  = var.orca_default_bucket
+  orca_restore_retrieval_type                          = var.orca_restore_retrieval_type
```

Comment:
This is not a required variable. It should be an optional variable and we should default to standard.

Reply:
moved